### ISSUE TITLE
Text Covering Reply, Edit, Delete buttons

### ISSUE THREAD

**Issue by cuttherug** [dokufreaks/plugin-discussion#326]
There isn't enough padding between the text and the buttons. It happens on the first line of nested replies, or after about three or four lines if leaving a long comment. Happening on mobile as well.

Thank you for the great plugin and thank you for any help.

![Screenshot 2022-05-04 183233 - Copy](https://user-images.githubusercontent.com/3887315/166852046-af5f81e0-bbc2-4b9d-8137-5e68f5813d2a.png)
.

**Comment by Klap-in** [comment]
Which template do you use? I cannot reproduce it using the 'dokuwiki' template.

**Comment by cuttherug** [comment]
I thought I was using the default template. I'll double check when I get
home. The site is definitiveversion.com. I'll send u a picture of my
settings or anything else u need.

**Comment by cuttherug** [comment]
Hi Gerrit,

It appears to only happen when logged in as and admin I think.

Also, yes I'm using the default template.

I attached a picture of some of my settings.

**Comment by Klap-in** [comment]
I've still no clue. (as guest I cannot see the buttons on your site).

Which version of DokuWiki are you using?

**Comment by cuttherug** [comment]
Hi again Gerrit,

I just updated to the latest version of Dokuwiki and it's still happening.

I'm open to discussing over Discord if you'd like.

**Comment by cuttherug** [comment]
![Screenshot_20220926-203254_Chrome](https://user-images.githubusercontent.com/3887315/192426626-772804a7-c10a-4626-9c79-bd8ba6c8c031.jpg)

**Comment by Klap-in** [comment]
I’m still searching how to reproduce it. Maybe there is interaction between different plugins.

**Comment by cuttherug** [comment]
Yeah, that could be a possibility I'll try to list everything I'm using
incase something sticks out to you.

**Comment by cuttherug** [comment]
This issues appears to still exist in the latest update, and turning off other plugins didn't seem to affect it. One thing I did notice is that it only appears on comments where you can see the "edit", "hide", and "delete" buttons. I think these can only be seen if it's a comment you leave or if you have admin privileges. Is there a site you know of using the discussion plugin so I can see how it works for someone else?

Thank you again.
